README.md

```diff
@@ -1 +1,38 @@
-# portfolio
\ No newline at end of file
+# Portfolio: setup and mobile version skeleton
+
+# Project Description
+
+> For the second milestone in the process of creating your portfolio website, you will: <
+
+> set up a new repository and prepare it for development using best practices (e.g. linters). <
+> build the first 2 sections of the mobile website using the template you chose in the previous step. <
+
+> This project is about setting up a mobile version skeleton <
+
+# Screenshot
+ [screenshot](./images/screenshot.png)
+ [screenshot](./images/screenshot2.png)
```

Comment:
Let's update the usage of the image as it's not displayed in the README kindly refer to [this link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images) how to add an image to the `README.md` file.

Reply:
Done